README.md

```diff
@@ -111,12 +113,25 @@ This is a schematic logging configuration to show log messages from input_nameA
 - `ovirt` type - `ovirt` input supports oVirt specific inputs.<br>
    For the details, visit [oVirt Support](../../design_docs/rsyslog_ovirt_support.md).
 
-- `remote` type - `remote` input supports receiving logs from the remote logging system over the network. This input type makes rsyslog a server.<br>
+- `relp` type - `relp` input supports receiving logs from the remote logging system over the network using relp.<br>
+  **available options**
+  - `port`: Port number Relp is listening to. Default to `20514`.
+  - `use_cert`: If true, key/certificates are used. Triplets {`ca_cert`, `cert`, `private_key`} and/or {`ca_cert_src`, `cert_src`, `private_key_src`} should be configured. Default to `true`.
```

Comment:
Is it possible to use RELP without TLS?  Is it possible to use RELP with TLS but without certs?

If you can use RELP without TLS, then maybe use `tls: true|false` instead of `use_cert`?  Unless it is somehow possible to use RELP with TLS but with no certs, but that would be strange.  I'm assuming RELP is using TLS for encryption, especially since it uses certs/keys.

Reply:
> Is it possible to use RELP without TLS? Is it possible to use RELP with TLS but without certs?
> 
> If you can use RELP without TLS, then maybe use `tls: true|false` instead of `use_cert`? Unless it is somehow possible to use RELP with TLS but with no certs, but that would be strange. I'm assuming RELP is using TLS for encryption, especially since it uses certs/keys.

Yes, we can use RELP without TLS. And `tls: true|false` should be better than `use_cert`. The thing is I found the set of RELP keys look similar to the one in the elasticsearch output and am hoping to have them in common, where use_cert is defined and used... Do you think it's too late to replace `use_cert` with `tls` in the elasticsearch output? I'd believe no one has even touched the output including the ovirt team. And the elasticsearch output is now available with the ovirt input only.

Reply:
I think add support for `tls: true|false` to the elasticsearch output AND keep support for `use_cert` - add documentation to the elasticsearch output saying that `use_cert` is deprecated in favor of `tls` and `use_cert` will be removed in the next minor release (e.g. 1.1.0)

Reply:
RELP is on the same level as TCP or UDP. The original idea of using protocol variable would fit here as well but that would make other this more complicated. While RELP is going to be a separate type is should look very similar to TCP, though.